dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -265,8 +367,37 @@ private void authorizedJobStopRestart(String submitter, long instanceId)
     																new Object[] {getRunAsUser(), instanceId},
```

Comment:
**IMPORTANT:** Needs to allow for path where this is a submitter and a group admin, like we did for read.